CMakeLists.txt

```diff
@@ -479,6 +479,25 @@ ELSE()
 ENDIF()
 
 
+# Find the libdap package so that Aspect can connect to the OPeNDAP servers
+# Author: Kodi Neumiller
+# Added: 10/16/18
+IF(LIBDAP)
+  LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}/cmake/modules")
+  MESSAGE(STATUS "cmake module path = ${CMAKE_MODULE_PATH}")
```

Comment:
remove this

Reply:
If I remove this then CMakeLists appears to not be able to run FindLIBDAP.cmake which is in the cmake/modules/ directory. If I remove the `LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}/cmake/modules")` then I have to put the FindLIBDAP.cmake file in the aspect directory, unless there is another way to do this.

Reply:
I can remove the `MESSAGE(STATUS "cmake module path = ${CMAKE_MODULE_PATH}")` if you want though, that was just there so the user knew where CMakeLists.txt was calling its other, external 'Find<package>.cmake' files.

Reply:
Yes, sorry, I meant the MESSAGE. I don't think it is particularly useful for the user to know.